app/controllers/users_controller.rb

```diff
@@ -0,0 +1,67 @@
+
+class UsersController < ApplicationController
+  skip_before_action :require_login,only: [:feed]
+
+  def show
+    @creators = current_user.creators
+  end
+
+  def feed
+    #user not logged in
+   if session[:user_id].nil?
+     redirect_to login_path
+   else
+     # check for new content from all the creators
+     current_user.creators.each do |creator|
+      # update creator from api if last updated more than 6 hours ago
+      if creator.last_updated.nil? || creator.last_updated < 6.hours.ago
+        creator.get_content
+      end
+     end
+     @contents = current_user.content.flatten
+     @contents = @contents.flatten.sort_by! { |content| content[:create_time] }.reverse
```

Comment:
You could condense these two lines a bit. You call `flatten` twice, but `flatten` accepts a parameter: the # of levels you want to flatten. In this case, you could use `flatten(2)` and only need to call it once.